test/Grpc.NetCore.HttpClient.Tests/AsyncServerStreamingCallTests.cs

```diff
@@ -176,5 +169,34 @@ public void ClientStreamReader_WriteWithInvalidHttpStatus_ErrorThrown()
 
             Assert.AreEqual("Bad gRPC response. Expected HTTP status code 200. Got status code: 404", ex.Message);
         }
+
+        [Test]
+        public async Task AsyncServerStreamingCall_NoContent_TrailersReturnedWithHeaders()
```

Comment:
nit: let's call this case "TrailersOnly" as that's how the case is called in the spec.